controllers/ApiController.php

```diff
@@ -284,43 +283,38 @@ public function init($extra) {
 			}
 		}
 		
-		$rateLimit = Zotero_API::getRateLimit([
-			'userID' => $this->userID,
-			'ip' => $_SERVER['REMOTE_ADDR']
-		]);
-		
-		$concurrencyLimit = Zotero_API::getConcurrencyLimit(['userID' => $this->userID]);
-		
+		list($rateLimit, $concurrencyLimit) = $this->limits();
```

Comment:
Can we move all of this into a separate method (e.g., `$this->initRateLimits()`)?